docker/Dockerfile.ci_adreno

```diff
@@ -25,4 +25,5 @@ COPY install/ubuntu_install_androidsdk.sh /install/ubuntu_install_androidsdk.sh
 RUN bash /install/ubuntu_install_androidsdk.sh
 ENV ANDROID_HOME=/opt/android-sdk-linux
 ENV ANDROID_NDK_HOME=/opt/android-sdk-linux/ndk/21.3.6528147
+ENV ANDROID_NDK_MAJOR=21
```

Comment:
I have doubts about explicit setting of the environment variable. As @lhez wrote, the `ANDROID_NDK_MAJOR` should be provided by NDK.

Reply:
It depends on cmake support. In our case it doesn't.